composer.json

```diff
@@ -269,6 +269,8 @@
         ],
         "integration-tests": [
             "@composer create-test-db",
+            "@composer install --working-dir=modules/ps_apiresources",
```

Comment:
Is it required to run composer install in the module? 🤔
Since the module is already a dependency from the core its classmap/autoload should already be in the general one.

Unless it ignores the dev dependencies?

Reply:
it seems that this is necessary in the bootstrap.php file. Also dependency problems arise without a composer installed

Reply:
`require_once dirname(__DIR__, 2) . '/vendor/autoload.php';` in `bootstrap.php` causes problem. We should see if this is really necessary, and if so, the conditioning.